package.json

```diff
@@ -108,7 +108,7 @@
             },
             {
                 "key": "ctrl+enter",
-                "when": "editorTextFocus && inputFocus && notebookEditorFocused || notebookCellListFocused",
```

Comment:
`notebookCellListFocused` is always true within the notebook widget, so the first half of this when clause shouldn't be necessary